Dockerfile

```diff
@@ -10,7 +10,11 @@ ENV PATH="/flutter/bin:${PATH}"
 
 # Running `flutter config --no-analytics` downloads the Dart SDK and
 # disables analytics tracking – which we always want
+# `1a1bbacf0d` maps to alpha release 0.0.8 @ June 13, 2017
+#    We hard wire the SHA to ensure Docker rebuilds
```

Comment:
There is a change the tag could be moved – but Docker wouldn't rebuild because the `RUN` content didn't change.

By hardwiring the SHA, we are ensured that the underlying state is not changing – unless we change it.